profile-widget/profilewidget2.cpp

```diff
@@ -1022,7 +1022,11 @@ void ProfileWidget2::wheelEvent(QWheelEvent *event)
 		scale(1.0 / zoomFactor, 1.0 / zoomFactor);
 		zoomLevel--;
 	}
+#if QT_VERSION >= QT_VERSION_CHECK(5, 14, 0)
```

Comment:
> Again, replacement was just recently added.

This is surprisingly unfriendly! You would think you wait a few years with deprecating a function after adding the alternative.

Reply:
This is all in preparation for Qt 6.0 - in the past they were far more reasonable with the speed of deprecating APIs.
I expect that it will be quite a while before we can stop supporting Qt 5.12 :-)